README.md

```diff
@@ -170,16 +177,17 @@ yarn install
 See the [help for `yarn link`](https://classic.yarnpkg.com/docs/cli/link) for
 more details about this.
 
-Running tests
-=============
+### Running tests
```

Comment:
Missed one

Reply:
I am not aware of a `---`-style title for h3 and below. Did I miss it?

Reply:
Ah i see, my bad